pages/user/[username]/repos/[repo]/issues/[issueNumber].js

```diff
@@ -381,14 +387,8 @@ class _Assignee extends Component {
   };
 
   render() {
-    const { classes, issue } = this.props;
-    const {
-      anchorEl,
-      open,
-      collaborators,
-      addAssignees,
-      removeAssignees
-    } = this.state;
+    const { classes, issue, collaborators } = this.props;
+    const { anchorEl, open, addAssignees, removeAssignees } = this.state;
     const AssigneeList =
```

Comment:
Variables should be start with capital names. Only for Components.

Reply:
done